### PR TITLE
build: automatically update renku-gateway dependency in renku

### PR DESCRIPTION
**Pull request by ableuler**
Note that the travis CLI did some auto-modifications to .travis-ci.yaml.

### CODE REVIEW

**Comment by rokroskar** [comment]
Instead of pushing directly to master and having to give the bot admin rights to the repo, can we instead push to a branch, say `/renku/auto-update/renku-gateway-<commit-sha>`?

cc @jachro: I think we should change this in `renku-graph` as well. I will make an action in the `renku` repo to automatically open PRs for these branches so there won't really be any need for manual intervention.

**Comment by ableuler** [comment]
> instead push to a branch, say `/renku/auto-update/renku-gateway-<commit-sha>`?

We agreed on using `/renku/auto-update/renku-gateway-<chart-version>` to distinguish tagged releases from regular ones.

File: .travis.yml [hunk 1]
```diff
@@ -1,54 +1,51 @@
...
 services:
-  - docker
-
+- docker
 language: python
-
 python:
-  - 3.7-dev
-
+- 3.7-dev
```
Comment (rokroskar) [inline .travis.yml]: I know you didn't just change this but why are we using a dev version?

File: .travis/travis-functions.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+function updateVersionInRenku() {
+  echo "Updating renku-gateway version to $CHART_VERSION in Renku's requirements.yaml"
+
+  # cloning the renku repo
+  cd || exit
+  git clone https://github.com/SwissDataScienceCenter/renku.git
+  cd renku || exit
+
+  # changing the version
+  sed -i "/- name: renku-gateway/{n;n;n;s/.*/  version: $CHART_VERSION/}" charts/renku/requirements.yaml
```
Comment (rokroskar) [inline .travis/travis-functions.sh]: in the future we should use `yq` to do this in a more human-readable way maybe

File: .travis.yml (hunk 1 above)
Comment (ableuler) [reply .travis.yml]: I don't remember, likely that's not necessary anymore.

File: .travis/travis-functions.sh (hunk 2 above)
Comment (ableuler) [reply .travis/travis-functions.sh]: Agreed.

**Pull request merged**